src/steps/generic.rs

```diff
@@ -1006,8 +1006,39 @@ pub fn run_dotnet_upgrade(ctx: &ExecutionContext) -> Result<()> {
     Ok(())
 }
 
+enum Hx {
+    Helix(PathBuf),
+    HxHexdump,
```

Comment:
This approach looks good to me, but I don't want to pull a tool that Topgrade does not support in. Can we just check if this `hx` is Helix or not. For example, `hx --help` contains "helix" and "text editor", can we make use of them?

Reply:
Fair point, changed it